packages/shared/locales/en.json

```diff
@@ -1800,6 +1800,36 @@
         "eventId": {
             "doesNotStartWith0x": "Event ID should start with '0x'",
             "insufficientLength": "Event ID should be 66 characters long"
+        },
+        "nft": {
+            "unsupportedUrl": {
+                "short": "Url not supported",
+                "long": "Url schema not supported"
+            },
+            "unsupportedFileType": {
+                "short": "Unsupported file type",
+                "long": "the file type is not currently supported"
```

Comment:
I would capitalize the first letter to match the other strings. But if you think about this string, it's actually the media type that is not supported since the function that throws this warning is checking whether the media type is `img` or `video`. Perhaps the string should actually be `This media type is not currently supported`

Reply:
And then the short string would change to `Unsupported media type`